daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -2761,6 +2761,14 @@ func NewDeviceSet(root string, doInit bool, options []string, uidMaps, gidMaps [
 				return nil, errors.New("dm.thinp_autoextend_threshold must be greater than 0 and less than 100")
 			}
 			lvmSetupConfig.AutoExtendThreshold = per
+		case "dm.libdm_log_level":
+			level, err := strconv.ParseUint(val, 10, 32)
+			if err != nil {
+				return nil, err
```

Comment:
Maybe, we should return a more expressive error here, as for instance `case "dm.thinp_autoextend_threshold":` does.